ShellBags/v2
# finish page setup and create mobile view

as the title says!

updates in this PR:
- addition of a new logo
- animations to some pages
- about and documentation pages have working links and some pages are filled with content, others require content from the app team
- mobile view integrated, resizing should no longer be an issue
- all pages now have comments 

steps to test:
- npm install/npm start from local
- otherwise use https://shellbags.github.io/v2/#/
- ensure all pages on the about and documentation page sidebars redirect properly
- check the site from mobile and ensure it looks good, or resize your browser
